internal/core/data/v2/controller/http/event.go

```diff
@@ -67,8 +70,9 @@ func (ec *EventController) AddEvent(w http.ResponseWriter, r *http.Request) {
 
 	// map Event models to AddEventResponse DTOs
 	var addResponses []interface{}
+	var successAddEventReqDTOs []requestDTO.AddEventRequest
 	for i, e := range events {
```

Comment:
I just realized constraint with this approach. All events being Added must be for the same profileName and deviceName. This constraint needs to be added to Swagger documentation for AddEvent.

Reply:
Swagger doc updated.